pkg/networkmanager/index.html

```diff
@@ -38,12 +38,12 @@ <h1 id="testing-connection-curtain-restoring" translate="yes">Restoring connecti
   </div>
 
   <div id="networking" class="pf-c-page" hidden>
-    <main  class="pf-c-page__main" tabIndex="-1">
+    <main class="pf-c-page__main" tabindex="-1">
```

Comment:
This commit is legit but will get redundant after the NM to react will be merged. That PR really removes all this HTML, so I don't care about merging this as well, the conflict will be straightforward.